app/src/main/java/org/jellyfin/androidtv/details/DetailsOverviewPresenter.kt

```diff
@@ -149,6 +151,16 @@ class DetailsOverviewPresenter : RowPresenter() {
 		}
 
 		if (item is PlayableItem) {
+			// (userData.getPlaybackPositionTicks() * 100.0 / itemDto.getRunTimeTicks()
+			val progress = (item.playbackPositionTicks * 100.0) / (item.durationTicks ?: 0)
```

Comment:
I don't like dividing by zero on purpose, moving the null-check outside feels cleaner to me.

Reply:
It would totally crash the application lol, I didn't' even remove the comment with the java code. Something went wrong when reviewing before commiting.